fe/fe-core/src/test/java/com/starrocks/utframe/UtFrameUtils.java

```diff
@@ -575,4 +588,113 @@ private static void unLock(Map<String, Database> dbs) {
             db.readUnlock();
         }
     }
+
+    public static void setUpForPersistTest() {
+        JournalReplayerHelper.setUp();
+        ImageHelper.setUpImageVersion();
+    }
+
+    public static void tearDownForPersisTest() {
+        JournalReplayerHelper.tearDown();
+    }
+
+    /**
+     * image helper is used to test if image is wrote correctly.
+     */
+    public static class ImageHelper {
```

Comment:
helper is conflict to the start_be.sh. `PseudoImage` may be a better name.

Reply:
done